notebooks/HST/beginner_zcut/requirements.txt

```diff
@@ -0,0 +1,4 @@
+astropy>=4.2.0
+matplotlib>=3.3.3
+numpy>=1.19.0
+astroquery>=0.4.2
```

Comment:
Just realized this line is causing an error, this is because 4.2 is not actually out yet, so what you want is:
`astroquery>=0.4.1.dev0`
(I'm pretty sure, if that doesn't work, we'll play around with it more)